experiment/scheduler.py

```diff
@@ -334,6 +336,21 @@ def get_nonpreemptible_starts(self) -> int:
         return get_started_trials(self.experiment_config['experiment']).filter(
             models.Trial.preemptible.is_(False)).count()
 
+    def _format_statistic_info(self, trial: models.Trial,
```

Comment:
This seems pretty overengineered for what we are trying to do. I would consider replacing this with something simpler.

Reply:
Let's just get this in and address any further feedback in a subsequent PR :)